test/functional/test_framework/wallet.py

```
@@ -185,9 +223,40 @@
+
+
+def address_to_scriptpubkey(address):
+    """Converts a given address to the corresponding output script (scriptPubKey)."""
+    payload, version = base58_to_byte(address)
+    if version == 140:  # testnet pubkey hash
+        return keyhash_to_p2pkh_script(payload)
+    elif version == 19:  # testnet script hash
+        return scripthash_to_p2sh_script(payload)
+    # TODO: also support other address formats
+    else:
+        assert False
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)